README.md

```diff
@@ -108,16 +108,16 @@ npm install @reactivex/rxjs@5.0.0-beta.1
 
 ### CDN
 
-For CDN, you can use [npmcdn](https://npmcdn.com):
+For CDN, you can use [unpkg](https://unpkg.com/):
 
 5.0.0-beta.1 - 5.0.0-beta.11:
 
-https://npmcdn.com/@reactivex/rxjs/dist/global/Rx.umd.js
+https://unpkg.com/@reactivex/rxjs/dist/global/Rx.umd.js
 
 
 5.0.0-beta.12 or higher:
 
-https://npmcdn.com/@reactivex/rxjs/dist/global/Rx.umd.js
+https://unpkg.com/@reactivex/rxjs/dist/global/Rx.umd.js
```

Comment:
Seems readme.md has not been updated while installation.md does (https://github.com/ReactiveX/rxjs/commit/82813f033366c595ef7e52b079ee54845502c8ef#diff-1db3204ec2751a8040f79683921d989dR102), from `Rx.umd.js` to `Rx.js`. it'd better to update once.